modules/components/package.json

```diff
@@ -49,12 +50,15 @@
     "eslint-plugin-jsx-a11y": "^6.1.2",
     "eslint-plugin-prettier": "^2.6.0",
     "eslint-plugin-react": "^7.6.1",
+    "eslint-plugin-react-hooks": "^4.3.0",
     "flow-bin": "^0.62.0",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
     "storybook-router": "^0.3.3"
   },
   "peerDependencies": {
-    "react": "^16.12.0",
-    "react-dom": "^16.3.0"
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0"
```

Comment:
making sure the peer dependencies match in version with the one used in implementation (and between the two, which I'm not sure why they were different here)